scripts/main/album.js

```diff
@@ -771,35 +838,62 @@ album.setSorting = function (albumID) {
 		);
 	};
 
-	let msg = lychee.html`
-		<div><p>
-			${sprintf(
-				lychee.locale["SORT_PHOTO_BY"],
-				`<span class="select">
-				<select id="sortingCol" name="sortingCol">
-					<option value=''>-</option>
-					<option value='created_at'>${lychee.locale["SORT_PHOTO_SELECT_1"]}</option>
-					<option value='taken_at'>${lychee.locale["SORT_PHOTO_SELECT_2"]}</option>
-					<option value='title'>${lychee.locale["SORT_PHOTO_SELECT_3"]}</option>
-					<option value='description'>${lychee.locale["SORT_PHOTO_SELECT_4"]}</option>
-					<option value='is_public'>${lychee.locale["SORT_PHOTO_SELECT_5"]}</option>
-					<option value='is_starred'>${lychee.locale["SORT_PHOTO_SELECT_6"]}</option>
-					<option value='type'>${lychee.locale["SORT_PHOTO_SELECT_7"]}</option>
-				</select>
-			</span>`,
-				`<span class="select">
-				<select id="sortingOrder" name="sortingOrder">
-					<option value=''>-</option>
-					<option value='ASC'>${lychee.locale["SORT_ASCENDING"]}</option>
-					<option value='DESC'>${lychee.locale["SORT_DESCENDING"]}</option>
-				</select>
-			</span>`
-			)}
-		</p></div>`;
+	const setAlbumSortingDialogBody = `
+		<form>
+			<div class="input-group compact">
+				<label for="sorting_dialog_column_select"></label>
+				<div class="select"><select name="sorting_col" id="sorting_dialog_column_select">
+					<option value=''>&mdash;</option>
+					<option value='created_at'/>
+					<option value='taken_at'/>
+					<option value='title'/>
+					<option value='description'/>
+					<option value='is_public'/>
+					<option value='is_starred'/>
+					<option value='type'/>
+				</select></div>
+			</div>
+			<div class="input-group compact">
+				<label for="sorting_dialog_order_select"></label>
+				<div class="select"><select name="sorting_order" id="sorting_dialog_order_select">
+					<option value=''>&mdash;</option>
+					<option value='ASC'/>
+					<option value='DESC'/>
+				</select></div>
+			</div>
+		</form>`;
+
+	/**
+	 * @param {ModalDialogFormElements} formElements
+	 * @param {HTMLDivElement} dialog
+	 * @returns {void}
+	 */
+	const initSetAlbumSortingDialog = function (formElements, dialog) {
+		formElements.sorting_col.labels[0].textContent = "Attribute";
```

Comment:
In theory yes. When I have been working on this PR, I recognized that probably 20% of all strings which should be localized are not yet enabled for localization. This one is not the only string. There are also still more labels, tooltips, etc.

This is not a regression.

I did not feel like fixing that problem within this PR, too. Because, I know myself. If I had started on that, then I would have gone the whole way which would have increased and extended this PR extremely. Hence, I decided to stick closely to the topic of this PR and to not touch anything else which crosses my path in between.

Reply:
I agree that you should not be fixing existing localization issues in this PR.

However, this particular case very much _does_ look like a regression to me, which is why I brought it up. In the code you replaced, there was `lychee.locale["SORT_PHOTO_BY"]`; you replaced it with hardcoded `"Attribute"` and `"Order"` strings...

Reply:
Fixed in https://github.com/LycheeOrg/Lychee-front/pull/322/commits/66619a4ba4a6c5039fb40a3cd8c4d7e627a00fdc.